apps/meteor/ee/app/livechat-enterprise/server/api/outbound.ts

```diff
@@ -0,0 +1,28 @@
+import { API } from '../../../../../app/api/server';
+import { isGETOutboundProviderParams } from '../outboundcomms/rest';
+import { outboundMessageProvider } from './lib/outbound';
+
+API.v1.addRoute(
+	'omnichannel/outbound/providers',
+	{ authRequired: true, validateParams: isGETOutboundProviderParams },
+	{
+		async get() {
+			const { type } = this.queryParams;
+
+			const providers = outboundMessageProvider.listOutboundProviders(type);
+			return API.v1.success({
+				providers,
+			});
+		},
+	},
+);
+
+API.v1.addRoute(
+	'omnichannel/outbound/providers/:id/metadata',
+	{ authRequired: true },
+	{
+		async get() {
+			return API.v1.success();
+		},
+	},
+);
```

Comment:
Would it be possible to use the new way of registering endpoints? Example: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/api/server/v1/custom-sounds.ts#L43

This way the endpoints will be already present in the Swagger docs 😃

Reply:
Also, we need to add e2e tests here, correct? 🤔

Reply:
Not for now, as the endpoints don't fully work (there's no way to use the providers yet)